source/install.txt

```diff
@@ -156,3 +156,7 @@ Next Steps
 
 Once you successfully install :binary:`mongosh`, learn how to
 :ref:`connect to your MongoDB deployment <mdb-shell-connect>`.
+
+MongoDB provides a programmatically accessible list of ``mongosh``
+`downloads <https://downloads.mongodb.com/compass/mongosh.json>`__ that
+can be accessed through your application.
```

Comment:
(question) 
Why are we including this at the bottom of the page? I know there is discussion of this in the jira comments, but I'm unclear on why this isn't either part of step 1 or mentioned in the pre-requisites. 

@jason-price-mongodb, @addaleax: Is the purpose of the programatic download to install mongosh? If so, what do you both think about reworking the download instructions so that we include this as a download option. 

If there is another reason for the download, cab we please provide a brief description of what it is and why this is this is the clear next step.

Reply:
> Is the purpose of the programatic download to install mongosh?

@sarah-olson-mongodb Yes, but not as the primary way for human users to do so, which I think our documentation is mainly targeting.

I’d definitely consider this to be more “something you can find in the installation docs” rather than “something we present to human users as a way to consider downloading mongosh”. If somebody wants a programmatic download, then they’ll be looking for _that_ specific thing.

Reply:
Thanks @addaleax, @jason-price-mongodb. 
This sounds ok. 

@jason-price-mongodb, I think we might flag this page as one in need of a fairly significant refactor. There is quite a bit going on in this page, and I find it difficult to navigate from time to time. In the refactor, I think we could consider a download section that walks people through the various options for the various OS. We could add this as a note, perhaps, there.

For this particular iteration, this addition seems fine.